docs/How-to-use-kaito-in-aks-arc.md

```diff
@@ -115,7 +115,6 @@ kind: Workspace
 metadata:
   name: workspace-falcon-7b
 resource:
-  instanceType: Standard_NC12s_v3

```

Comment:
If the preferred node cannot be found, a fallback `instanceType` is used, defaulting to `Standard_NC12s_V3`. We retain this field for transparency.

Reply:
I think a default sku can be added once gpuprovisioner supports arc RP apis. For now, it is ok to remove it.